spec/intl.html

```diff
@@ -1791,6 +1772,30 @@ <h1>
             In a _calendar_ such as *"japanese"* where eras do not start and end at year and/or month boundaries, note that the returned List should contain *"era"* and *"eraYear"* if _keys_ contains *"day"*, *"month"*, or *"monthCode"* (not only if it contains *"era"*, *"eraYear"*, or *"year"*, as in the example above) because it's possible for changing the day or month to cause a conflict with the era.
           </emu-note>
         </emu-clause>
+
+        <emu-clause id="sec-temporal-calendarresolvefields" type="abstract operation">
+          <h1>
+            CalendarResolveFields (
+              _calendar_: a String,
+              _fields_: an Object,
+              _type_: ~date~, ~year-month~, or ~month-day~,
+            ): either a normal completion containing ~unused~ or a throw completion
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>
+              It performs implementation-defined processing to validate that _fields_ (which describes a date in the built-in calendar identified by _calendar_) is sufficiently complete to satisfy _type_ and not internally inconsistent, and merges data that can be represented in multiple forms into standard properties of _fields_ (for example, merging *"month"* and *"monthCode"* into *"month"* and merging *"era"* and *"eraYear"* into *"year"*).
+            </dd>
+          </dl>
+          <p>
+            The operation throws a *TypeError* exception if the properties of _fields_ are internally inconsistent within the calendar or insufficient to identify a unique instance of _type_ in the calendar. For example:
+          </p>
+          <ul>
+            <li>If _type_ is ~date~ or ~month-day~ and *"day"* has the usual interpretation in the calendar and the corresponding value is *undefined*.</li>
```

Comment:
> has the usual interpretation in the calendar

I found this phrase a bit confusing. Should we either clarify or somewhere else define what "usual interpretation in the calendar" means?

Reply:
Updated.